app/views/guidance/selecting-sites-for-development-for-your-local-plan/stage-2-categorising-and-assessing-sites.html

```diff
@@ -0,0 +1,768 @@
+                            stroke-width="3"
+                          ></line>
+                        </svg>
+                      </button>
+                    </div>
+                  </div>
+                </div>
+              </form>
+            </div>
+          </div>
+        </header>
+        <div
+          data-module="gem-track-click ga4-link-tracker"
+          class="gem-c-breadcrumbs govuk-breadcrumbs gem-c-breadcrumbs--border-bottom"
+          data-gem-track-click-module-started="true"
+          data-ga4-link-tracker-module-started="true"
+        >
+          <ol class="govuk-breadcrumbs__list">
+            <li class="govuk-breadcrumbs__list-item">
+              <a
+                data-track-category="breadcrumbClicked"
+                data-track-action="1"
+                data-track-label="/guidance/selecting-sites-for-development-for-your-local-plan"
+                data-track-options='{"dimension28":"1","dimension29":"Contents"}'
+                data-ga4-link='{"event_name":"navigation","type":"breadcrumb","index_link":"1","index_total":"1"}'
+                class="govuk-breadcrumbs__link"
+                href="/guidance/selecting-sites-for-development-for-your-local-plan"
+                >Contents</a
+              >
+            </li>
+          </ol>
+        </div>
+        <main role="main" id="content" class="manual-section" lang="en">
+          <span id="Top"></span>
+          <div id="manuals-frontend" class="manuals-frontend-body">
+            <div class="govuk-grid-row">
+              <div class="manual-body">
+                <article aria-labelledby="section-title">
+                  <div class="govuk-grid-column-full">
+                    <h1
+                      id="section-title"
+                      class="gem-c-heading govuk-heading-l govuk-!-margin-bottom-4"
+                    >
+                      Stage 2 - Categorising and assessing sites
+                    </h1>
+                  </div>
+
+                  <div class="govuk-grid-column-two-thirds">
+                    <p class="gem-c-lead-paragraph">
+                      Categorising your list of potential sites and performing a
+                      detailed assessment of their suitability, availability and
+                      achievability
+                    </p>
+                  </div>
+
+                  <div class="govuk-grid-column-two-thirds">
+                    {{ govukAccordion({ id: "accordion-default", items: [ {
+                    heading: { text: "Categorisation" }, content: { html: '
+                    <h3 id="links">Links</h3>
+                    <p>
+                      <a href="">Why it’s useful to categorise sites</a>
+                    </p>
+                    <p><a href="">How you could categorise sites</a></p>
+                    <p>
+                      <a href="">Deciding what you can build on your sites</a>
+                    </p>
```

Comment:
Do we know where these are supposed to link to?

Reply:
(see the comment thread here: https://github.com/digital-land/local-plans-home/pull/20#discussion_r1538788698)